lib/translator/translation.cc

```diff
@@ -69,8 +73,10 @@ ApiStatus Translation::Complete(
   scsi_defs::OpCode opc = static_cast<scsi_defs::OpCode>(scsi_cmd_[0]);
   switch (opc) {
     case scsi_defs::OpCode::kInquiry:
+      pipeline_status_ = InquiryToScsi(scsi_cmd_, buffer, GetNvmeCmds());
       return ApiStatus::kSuccess;
   }
+  return ApiStatus::kFailure;
```

Comment:
Is an unsupported SCSI opcode a misuse of the API? What's the caller expected to do to avoid that? Would it be practical to return a SCSI error rather than an API error, so the caller doesn't have to worry about such things?

Reply:
We definitely have plans for SCSI responses based on the `pipeline_status_` variable but I believe its not implemented yet. @harloff-g can comment in more detail

Reply:
IMO this should be structured consistent with those plans, even if they're just a TODO for now.

Reply:
i will change this to reflect what we decide to do on the `gen-translator` branch